docs/src/main/sphinx/language/types.rst

```diff
@@ -262,11 +262,27 @@ The following examples illustrate the behavior::
 Instant in time that includes the date and time of day with ``P`` digits of
 precision for the fraction of seconds and with a time zone. Values of this
 type are rendered using the time zone from the value.
-Time zones are expressed as the numeric UTC offset value::
+Time zones can be expressed in the following ways:
+
+* ``UTC``, with ``GMT``, ``Z``, or ``UT`` usable as aliases for UTC.
+* ``+hh:mm`` or ``-hh:mm`` with ``hh:mm`` as an hour and minute offset from UTC.
+  Can be written with or without ``UTC``, ``GMT``, or ``UT`` as an alias for
+  UTC.
+* An `IANA time zone name <https://www.iana.org/time-zones>`_.
+
+The following examples demonstrate these syntax options::
```

Comment:
nit: the examples are not exhaustive list obviously

```suggestion
The following examples demonstrate some of these syntax options::
```

Reply:
Good point, added